core/src/main/scala/com/salesforce/op/evaluators/EvaluationMetrics.scala

```diff
@@ -176,6 +176,22 @@ object RegressionEvalMetrics extends Enum[RegressionEvalMetric] {
 }
 
 
+/**
+ * Forecast Metrics
+ */
+sealed abstract class ForecastEvalMetric
+(
+  val sparkEntryName: String,
+  val humanFriendlyName: String
+) extends EvalMetric
+
+
+object ForecastEvalMetrics extends Enum[RegressionEvalMetric] {
+  val values: Seq[RegressionEvalMetric] = findValues
+  case object sMAPE extends RegressionEvalMetric("smape", "symmetric Mean Absolute Percentage Error")
```

Comment:
lets rename to `SMAPE` and lowercase the human friendly name to `symmetric mean absolute percentage error` (as we do with all other metrics)